### DISCUSSION TITLE
Display data from arrays not objects

### DISCUSSION THREAD

**Discussion by lkoehl** [TanStack/table#2853] [Q&A] [answered]
Hey guys, 
I have my data in just arrays and would like to render it out with react-table. I have taken a look at the accessor but it doesnt do the trick for me. [Here](https://codesandbox.io/s/infallible-pond-pntw1?fontsize=14&hidenavigation=1&theme=dark&file=/src/App.js) is a demo of my problem. I hope you guys can help me out.

**Comment by Kamino0** [comment]
Hello, did you find a solution?

**Reply by lkoehl** [reply]
Not a satisfying one. My current solution is to iterate over the array, building an object out of it which i feed into react-table.

**Comment by JulesBlm** [comment, marked as answer]
This seems to be a simple spelling error. You're missing an s in "accessor" in your CodeSandbox example, if you fix that it works.

**Reply by lkoehl** [reply]
Amazing.
